code/src/components/OverLay.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+import { SharedStyles } from "./SharedStyles";
+
+export const OverLay = styled.div`
+  ${SharedStyles}
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: end;
+  opacity: 0;
+  transition: opacity 0.2s ease;
+
```

Comment:
Could it be that you need a scroll position something to help the positioning be relative to when you have scrolled down in the movielist?